README.md

```diff
@@ -55,6 +55,20 @@ The following tools are required for development:
         brew install mongodb-community@4.2
         brew services start mongodb-community@4.2
 
+- To support the parsing of messages from RabbitMQ instead of via SFTP, both
+  RabbitMQ and Redpanda must be available.  Running these from Docker is highly
+  recommended.  Follow the instructions under the Docker section of this
+  document to bringing up the dependencies and ensure these are available.
+
+### Additional Dependencies
+
+If you intend to test/develop/use the test data generation functionality of
+Crawler at the `/v1/cherrypick-test-data` endpoint, you will also need to be
+running a local instance of [Baracoda](https://github.com/sanger/baracoda).  The
+default port of Baracoda is 8000, but if you need to use a different port,
+change the port specified for Baracoda in the `crawler/config/defaults.py` file
+in this repository.
```

Comment:
I think I would include the name of the constant too, so they know what to change; and I will remove the 8000 and only say ' if need to use a different port than the default...' in case that is modified because it seems a comment that can get out of date very easily.